src/devtools/useAtomsDevtools.ts

```diff
@@ -52,11 +53,29 @@ const getDevtoolsState = (atomsSnapshot: AtomsSnapshot) => {
   }
 }
 
-export function useAtomsDevtools(name: string, scope?: Scope) {
+interface DevtoolsOptions {
+  scope?: Scope
+  enabled?: boolean
+}
+
+export function useAtomsDevtools(name: string, options?: DevtoolsOptions): void
+/*
+ * @deprecated Please use object options (DevtoolsOptions)
+ */
+export function useAtomsDevtools(name: string, options?: Scope): void
+
+export function useAtomsDevtools(
+  name: string,
+  options?: DevtoolsOptions | Scope
+) {
+  if (typeof options !== 'undefined' && typeof options !== 'object') {
+    options = { scope: options }
+  }
+  const { enabled, scope } = options || {}
   const ScopeContext = getScopeContext(scope)
   const { s: store, w: versionedWrite } = useContext(ScopeContext)
 
-  if (!store[DEV_SUBSCRIBE_STATE]) {
+  if (!store[DEV_SUBSCRIBE_STATE] && enabled && __DEV__) {
```

Comment:
I still want the early return.
Which test fails? I mean it should only fail if `enabled === false`, right? It should be a new test.

```suggestion
  if (enabled === false) {
    return
  }
  if (!store[DEV_SUBSCRIBE_STATE]) {
```

Reply:
I completely forgot about this being a hook. We can't early return. That's really my bad. So sorry to have your time wasted. Added a commit: dc485b7